content/posts/2023-08-28_the-upside-of-impostor-syndrome.md

```diff
@@ -0,0 +1,22 @@
++++ 
+date = 2023-08-28T22:39:33+02:00
+title = "The Upside of Impostor Syndrome"
+description = "Remember that the origins of Impostor Syndrome comes from a healthy place."
+slug = "the-upside-of-impostor-syndrome"
++++
+The upside of impostor syndrome is that it means that you have the capabitity
+of self-reflection — the possibility to look at yourself with critical eyes.
+This is actually a healthy thing in small doses. I am sure most of us have
+worked with a colleague who say “this is the only way” and “we have tried that,
+it will never work”. These are narrow-minded engineers closed off to reconsider
+alternative solutions and explanations - they lack a healthy sense of
+self-reflection.
+
```

Comment:
```suggestion
The upside of [Impostor Syndrome] is that it means that you have the capabitity
of self-reflection — the possibility to look at yourself with critical eyes.
This is actually a healthy thing in small doses. I am sure most of us have
worked with a colleague who say “this is the only way” and “we have tried that,
it will never work”. These are narrow-minded engineers closed off to reconsider
alternative solutions and explanations - they lack a healthy sense of
self-reflection.

[impostor-syndrome]: https://en.wikipedia.org/wiki/Impostor_syndrome

```